internal/tools/api-version-lint/main.go

```diff
@@ -0,0 +1,117 @@
+package main
```

Comment:
i might want to make this tool something more explicit like `preview-api-version-lint[er]` so its more descriptive in it focuses on preview apis